sphinx/util/jsdump.py

```diff
@@ -192,3 +194,21 @@ def loads(x: str) -> Any:
 
 def load(f: IO) -> Any:
     return loads(f.read())
+
+
+deprecated_alias(
+    'sphinx.util.jsdump',
+    {
+        'dumps': dumps,
+        'dump': dump,
+        'loads': loads,
+        'load': load,
+    },
+    RemovedInSphinx70Warning,
+    {
+        'dumps': 'json.dumps',
+        'dump': 'json.dump',
+        'loads': 'json.loads',
+        'load': 'json.load',
+    }
+)
```

Comment:
Please use `warnings.warn()` instead of `deprecated_alias()` for this case. `deprecated_alias()` lets developers know the specified attributes and functions has been deprecated and will be removed on accessing them.

In this case, we'd like to deprecate the whole of the module. So it's better to warn the deprecation on importing.
```suggestion
warnings.warn('sphinx.util.jsdump has been deprecated. Please use "json" instead.', RemovedInSphinx70Warning)
```

Reply:
@tk0miya done

A